core/creators/themedui.js

```diff
@@ -253,12 +253,12 @@ CKEDITOR.replaceClass = 'ckeditor';
 	 *
 	 *		editor.resize( 900, 300 );
 	 *
-	 *		editor.resize( '100%', 450, true );
+	 *		editor.resize( '5in', 450, true );
 	 *
 	 * @param {Number/String} width The new width. It can be an integer denoting a value
-	 * in pixels or a CSS size value with unit.
+	 * in pixels or a CSS size value with absolute unit.
 	 * @param {Number/String} height The new height. It can be an integer denoting a value
-	 * in pixels or a CSS size value with unit.
+	 * in pixels or a CSS size value with absolute unit.
```

Comment:
I'm wondering about being more explicit here, as previously these docs have been incorrect. Maybe instead of adding information about absolute units for each param, we could add more generic information right below `Resizes the editor interface`?

```
Resizes the editor interface.

**Note:** Since 4.14.1 this method accepts only numeric or absolute CSS length units.
```